apps/admin/app/(auth)/reset-password/page.tsx

```diff
@@ -0,0 +1,12 @@
+import { Metadata } from "next";
+import { ResetPasswordForm } from "./_components/form";
+
+
+export const metadata: Metadata = {
```

Comment:
```suggestion
export const metadata: Metadata = {
```
shall this be added as global object?